app/decorators/plant_trial_submission_decorator.rb

```diff
@@ -43,7 +43,7 @@ def plant_population
 
   def trait_descriptors
     tds = object.content.step02.trait_descriptor_list.try(:select, &:present?) || []
-    numeric_tds = tds.select { |td| td.to_s.match /\A\d+\z/ }
+    numeric_tds = tds.select { |td| td.to_s.match(/\A\d+\z/) }
```

Comment:
I understand `numeric_tds` refer in fact to `existing` TDs that are already present in the DB. If this is indeed the case, can we change this naming a bit? Since we use 'new_trait_de....; elsewhere, can we call the existing ones `old`?

BTW what happens to that match if one calls her new TD with a number?